public/css/common.less

```diff
@@ -303,8 +303,11 @@ form[name="form_confirm_removal"] {
 .affected-objects {
   display: inline-flex;
   align-items: baseline;
-  background-color: @state-critical;
-  color: @text-color-inverted;
+  color: @state-critical;
   padding: 0 0.25em;
   .rounded-corners();
```

Comment:
Please remove this as there is now no background and this has no visible effect.